modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java

```diff
@@ -1012,14 +1012,23 @@ private boolean _hasLiferayNpmScripts12Dependency(File packageJSONFile) {
 		Map<String, Object> devDependencies =
 			(Map<String, Object>)packageJSONMap.get("devDependencies");
 
-		if ((devDependencies == null) ||
-			!devDependencies.containsKey("liferay-npm-scripts")) {
+		if (devDependencies == null) {
+			return false;
+		}
+
+		String dependencyName = null;
 
+		if (devDependencies.containsKey("@liferay/npm-scripts")) {
+			dependencyName = "@liferay/npm-scripts";
+		}
+		else if (devDependencies.containsKey("@liferay/npm-scripts")) {
+			dependencyName = "liferay-npm-scripts";
+		}
```

Comment:
Is this wrong? :thinking: (the two ifs)

Reply:
You have passed the test. Well done, my young ~padawan~ kung fu fighter.

![wise-man](https://user-images.githubusercontent.com/7074/94692892-5fc69f80-0333-11eb-8e97-99387ce9f173.gif)